CHANGELOG-3.0.md

```diff
@@ -37,6 +37,7 @@ To get the diff between two versions, go to https://github.com/symfony/symfony/c
  * bug #18761  [Form] Modified iterator_to_array's 2nd parameter to false in ViolationMapper (issei-m)
  * bug #18745 [MonologBridge] Uninstallable together with symfony/http-kernel in 3.0.6 (ymc-dabe)
  * bug #18737 [Debug] Fix fatal error handlers on PHP 7 (nicolas-grekas)
+ * feature #19086 [FrameworkBundle] added configurable paths for validation files 
```

Comment:
This should be removed. I was talking about the CHANGELOG in FrameworkBundle. This one is auto-generated when I make a release. Sorry about the confusion.